.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.2
```

Comment:
i think i might take this one out, this is only my local version of ruby in any case